setup.py

```diff
@@ -31,11 +31,6 @@ def is_exe(fpath):
     print("Unfortunately, we do not support your platform %s" % sys.platform)
     sys.exit(1)
 
-if which('dexdump') is None:
-    print("Unable to find dexdump executable, please install it.")
-    print("On Debian-like OS, run sudo apt-get install dexdump")
-    sys.exit(1)
```

Comment:
Since you removed dexdump and the call to `which` you can remove the function at the top of the file.